templates/todomvc/src/test.js

```diff
@@ -0,0 +1,26 @@
+describe("Token", function() {
+    let near;
+    let contract;
+    let alice;
+    let bob = "bob.near";
+    let eve = "eve.near";
+  
+    beforeAll(async function() {
+      const config = await nearlib.dev.getConfig();
+      console.log("nearConfig", config);
+      near = await nearlib.dev.connect();
+      alice = nearlib.dev.myAccountId;
+      contract = await near.loadContract(config.contractName + "test", {
```

Comment:
let's pass proper `contractName` in config in near fiddle api

I think we can pass some query parameter to near pages, so that we open something like:

`https://app.near.ai/fiddleId/test.html?contractName=test-123213132312`